### PR TITLE
Make markdown supported tags configurable with sane defaults

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR make markdown allowed tags configurables and provide larger default than bleach defaults.

### CODE REVIEW

File: udata/settings.py [hunk 1]
```diff
@@ -56,6 +56,34 @@ class Defaults(object):
...
     OAUTH2_PROVIDER_ERROR_ENDPOINT = 'oauth-i18n.oauth_error'
 
+    MD_ALLOWED_TAGS = [
+        'a',
+        'abbr',
+        'acronym',
+        'b',
+        'br',
+        'blockquote',
+        'code',
```
Comment (davidbgk) [inline udata/settings.py]: Can you add a test combining `<code>` and urls within? I'm pretty sure urls will be linkified within it and that's not what we expect.

Comment (noirbizarre) [inline udata/settings.py]: OK, I will

Comment (noirbizarre) [inline udata/settings.py]: Added in the last commits

File: udata/tests/frontend/test_markdown.py [hunk 2]
```diff
@@ -53,6 +53,30 @@ def test_markdown_linkify(self):
...
+        '''Markdown filter should transform emails to anchors'''
+        text = 'coucou@cmoi.fr'
+        with self.app.test_request_context('/'):
+            result = render_template_string('{{ text|markdown }}', text=text)
+            parsed = parser.parse(result)
+            el = parsed.getElementsByTagName('a')[0]
+            self.assertEqual(el.getAttribute('href'), 'mailto:coucou@cmoi.fr')
+            self.assertEqual(el.firstChild.data, 'coucou@cmoi.fr')
+
+    def test_markdown_linkify_wihtin_pre(self):
```
Comment (davidbgk) [inline udata/tests/frontend/test_markdown.py]: typo for "within" + *not* linkify?

File: udata/tests/frontend/test_markdown.py [hunk 3]
```diff
@@ -53,6 +53,30 @@ def test_markdown_linkify(self):
...
+        '''Markdown filter should not transform urls into <pre> anchors'''
+        text = '<pre>http://example.net/</pre>'
+        with self.app.test_request_context('/'):
+            result = render_template_string('{{ text|markdown }}', text=text)
+            self.assertEqual(result.strip(), '<pre>http://example.net/</pre>')
+
+    def test_markdown_linkify_email_wihtin_pre(self):
```
Comment (davidbgk) [inline udata/tests/frontend/test_markdown.py]: same here

**Comment by davidbgk** [comment]
r+wc

**Pull request merged**